data/brands/shop/hardware.json

```diff
@@ -18,6 +18,23 @@
         "shop": "hardware"
       }
     },
+    {
+      "displayName": "Do it Best",
```

Comment:
I'm not sure about the name here. All the locations around me are signed and referenced as XYZ Do it Best Hardware